CiarnuroRT/webapp/index.html

```diff
@@ -116,8 +116,8 @@
         <div class="slide invisible" id="gameTimer">
             <div class="upper">
                 <div class="content">
-                    <div class="title">Turno <span id="roundNumber"></span> - <span id="gtDate"></span></div>
-                    <div class="banner"><span id="playerAndCharacterName"></span><br/><span id="gtTeamName"></span></div>
+                    <div class="title">Turno di gioco <span id="roundNumber"></span> - <span id="gtDate"></span></div>
+                    <div style="text-align:center;font-size:1.4em;margin-bottom: 0.2em;" id="playerInfo"><img id="profilePic" style="position:relative;left:0.5em;border-radius:50%;height:4em;width:4em;border:0.2rem solid #404040;"/><br><span id="playerAndCharacterName" style="font-weight:bold;"></span><br/><span id="gtTeamName"></span></div>
```

Comment:
Inline CSS should only be used for small variations of the style, move this style in the CSS file